lib/sneakers_handlers/exponential_backoff_handler.rb

```diff
@@ -0,0 +1,121 @@
+module SneakersHandlers
+  class ExponentialBackoffHandler
+    attr_reader :queue, :channel, :options, :delays
+
+    def initialize(channel, queue, options)
+      @queue = queue
+      @channel = channel
+      @options = options
+      @delays = options[:delays] || [1.second, 10.seconds, 1.minute, 10.minutes]
+
+      Array(@options[:routing_key]).each do |key|
+        queue.bind(primary_exchange, routing_key: key + ".*")
```

Comment:
This works fine for us, but I think we should make it clear that this handler will work just for `topic` exchanges.